index.bs

```diff
@@ -798,7 +805,8 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
     1. Abort these steps.
   1. Set |transport|.{{[[State]]}} to `"connected"`.
   1. Set |transport|.{{[[Session]]}} to |session|.
-  1. Set |transport|'s {{[[Reliability]]}} to `"supports-unreliable"`.
+  1. If the connection is an HTTP/3 connection, set |transport|'s {{[[Reliability]]}} to `"supports-unreliable"`.
+  1. If the connection is an HTTP/2 connection [[WEB-TRANSPORT-HTTP2]], set |transport|'s {{[[Reliability]]}} to `"reliable-only"`.
```

Comment:
Text looks great, just double checking that we're clear that datagram delivery will still be unreliable (can take that to a separate issue, though).

Reply:
(Thank you @martinthomson!)